### DISCUSSION TITLE
Which Chart library are you using with shadcn and React?

### DISCUSSION THREAD

**Discussion by enszrlu** [shadcn-ui/ui#4133] [General]
I am planning to add a dashboard to my app. I have been using shadcn for everything so far so I want charts to follow same design style.

Which library are you guys using or suggesting?

**Comment by simovicaleksa** [comment]
I highly suggest you take a look at [ReCharts](https://recharts.org/en-US/examples/SimpleBarChart).

It is the best library for charts in the react ecosystem imo. You can style the charts however you'd like and the DX is great. On top of that it is being actively maintained.

If you are interested in other options take a look at: [ECharts](https://echarts.apache.org/examples/en/index.html#chart-type-line), [React Vis](https://uber.github.io/react-vis/) and [Nivo](https://nivo.rocks/components/).

**Reply by enszrlu** [reply]
Recharts is on my top 4 at the moment.
Others are Tremor, Apexcharts and antcharts.
